vello/src/recording.rs

```diff
@@ -168,7 +168,10 @@ impl Recording {
     /// Dispatch a compute shader where the size is determined dynamically.
     /// The `buf` argument contains the dispatch size, 3 `u32` values beginning
     /// at the given byte `offset`.
-    #[allow(unused)]
+    #[cfg_attr(
+        not(feature = "debug_layers"),
+        expect(unused, reason = "Currently only used by the debug layers")
+    )]
```

Comment:
This is `pub`, so why is it needing to be marked as `unused`?

Reply:
Hmm, actually it wasn't unused. Sorry I didn't analyse this fully.